frontend/src/views/app-store/installed/index.vue

```diff
@@ -642,4 +642,9 @@ onUnmounted(() => {
     max-height: 500px;
     overflow-y: auto;
 }
+.d-button {
+    .el-button + .el-button {
+        margin-left: 0;
+    }
+}
 </style>
```

Comment:
It appears that there's no specific code difference you mentioned or an issue to discuss regarding regularity, functionality, or optimizations. It is important to note though, without further context (such as a previous commit history, test cases, etc), these discussions are generally speculative and can't conclusively determine whether something is wrong or not with accuracy.